src/runner.rs

```diff
@@ -1263,8 +1260,47 @@ impl ValueStack {
 		Ok(())
 	}
 
+	#[inline]
+	fn extend(&mut self, values: &[RuntimeValue]) -> Result<(), TrapKind> {
+		let len = cmp::min(values.len(), self.buf.len() - self.sp);
```

Comment:
Consider giving a name to `self.buf.len() - self.sp` expression

Reply:
sure.